fastparse/shared/src/main/scala/fastparse/sequence/VectorReprOps.scala

```diff
@@ -0,0 +1,34 @@
+package fastparse.sequence
+import fastparse.utils._
+import scala.reflect.ClassTag
+
+case class VectorReprOps[T]()(implicit evidence: ClassTag[T]) extends fastparse.utils.ReprOps[T, Vector[T]] {
+  type U = Vector[T]
```

Comment:
`U` is a bit of an odd name here. Could we call it `type VecT`? Slightly longer but much more clear